dg/media/analytics/loop/configs/TabsConfig.ts

```diff
@@ -0,0 +1,16 @@
+export const tabsConfig = {
```

Comment:
Shouldn't this file be in training module?

Reply:
done. removed. was unsed.